### DISCUSSION TITLE
What is requirement of flet

### DISCUSSION THREAD

**Discussion by BtAbdul** [flet-dev/flet#1858] [Q&A]
### Question

Hello,

I want to know list of python packages 📦 (module) list that required for flet?

Technically requirements.txt file sometimes like that

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by SKbarbon** [comment]
@BtAbdul check this:

https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet/pyproject.toml
